lms/static/js/spec/commerce/receipt_spec.js

```diff
@@ -0,0 +1,182 @@
+                                "price": "10.00",
+                                "expires": null,
+                                "is_available_to_buy": true,
+                                "id": 123,
+                                "structure": "child"
+                            },
+                            "line_price_excl_tax": "10.00",
+                            "description": "dummy description",
+                            "title": "dummy title",
+                            "quantity": 1
+                        }
+                    ],
+                    "number": "EDX-123456",
+                    "date_placed": "2016-01-01T01:01:01Z",
+                    "currency": "USD",
+                    "total_excl_tax": "10.00"
+                };
+                providerResponseData = {
+                        "id": "edx",
+                        "display_name": "edX",
+                        "url": "http://www.edx.org",
+                        "status_url": "http://www.edx.org/status",
+                        "description": "Nothing",
+                        "enable_integration": false,
+                        "fulfillment_instructions": "",
+                        "thumbnail_url": "http://edx.org/thumbnail.png"
+                };
+                courseResponseData = {
+                    "id": "course-v1:edx+dummy+2015_T3",
+                    "name": "receipt test",
+                    "category": "course",
+                    "org": "edx",
+                    "run": "2015_T2",
+                    "course": "CS420",
+                    "uri": "http://test.com/api/course_structure/v0/courses/course-v1:edx+dummy+2015_T3/",
+                    "image_url": "/test.jpg",
+                    "start": "2030-01-01T00:00:00Z",
+                    "end": null
+                };
+
+            });
+
+            it("sends analytic event when verified receipt is rendered", function() {
+                loadVerifiedReceiptFixture();
+                loadReceiptFixture();
+                mockRender();
+                expect(window.analytics.track).toHaveBeenCalledWith(
+                    "Completed Order",
+                    {
+                        orderId: "EDX-123456",
+                        total: "10.00",
+                        currency: "USD"
+                    }
+                );
+
+            });
+
+            it("sends analytic event when non verified receipt is rendered", function() {
+                loadNonVerifiedReceiptFixture();
+                loadReceiptFixture();
+                mockRender();
+                expect(window.analytics.track).toHaveBeenCalledWith(
+                    "Completed Order",
+                    {
+                        orderId: "EDX-123456",
+                        total: "10.00",
+                        currency: "USD"
+                    }
+                );
+
+            });
+
+            it("test render receipt", function() {
```

Comment:
Nit: the test name should read like a sentence starting with ``it``, so something like:

    it('renders a receipt correctly', function() {